packages/storage/amplify_storage_s3/lib/amplify_storage_s3.dart

```diff
@@ -18,6 +18,7 @@ library amplify_storage_plugin;
 import 'dart:io';
 
 import 'package:amplify_core/amplify_core.dart';
+import './src/S3PrefixResolver/amplify_storage_s3_prefix_resolver.dart';
```

Comment:
Organize

Reply:
Also naming: `S3PrefixResolver` -> `s3_prefix_resolver`, and use package imports. We are enforcing this on next with [always_use_package_imports](https://dart-lang.github.io/linter/lints/always_use_package_imports.html). Good idea to use it everywhere.

Reply:
alright I can make that change.  Are you sure @Jordan-Nelson as this would be inconsistent with the existing naming existing for main: 

s3_prefix_resolver
S3DownloadFile
S3GetUrl
S3List
S3UploadFile